_shared/project/.github/workflows/ci.yml

```diff
@@ -14,14 +14,21 @@ jobs:
     needs: tests
     {% endif %}
     runs-on: ubuntu-latest
-    {% if cookiecutter.get("db") == "yes" %}
+    {% if cookiecutter.get("postgres") == "yes" or include_exists(".github/workflows/ci/services.yml") %}
```

Comment:
Add the `services` section to `ci.yml` if _either_ `"postgres"` is `"yes"` in `cookiecutter.json` _or_ `.cookiecutter/includes/.github/workflows/ci/services.yml` exists.